.github/workflows/tests.yml

```diff
@@ -56,3 +67,4 @@ jobs:
         run: bats test
         env:
           GITHUB_API_TOKEN: ${{ github.token }}
+          SKIP_NUSHELL_TESTS:  "${{ matrix.os == 'ubuntu-18.04' && 'YES' || 'NO' }}"
```

Comment:
The `|| 'NO'` is probably unnecessary since you explicitly check for `YES` in the test code.